README.md

```diff
@@ -38,6 +38,7 @@ framework when using Sauce Labs.
         - [Examples](webdriverio/webdriver/examples)
             - [WebdriverIO and CucumberJS](webdriverio/webdriver/examples/cucumberjs)\
               ![WebdriverIO - Webdriver - Examples - CucumberJS](https://github.com/saucelabs-training/demo-js/workflows/WebdriverIO%20-%20Webdriver%20-%20Examples%20-%20CucumberJS/badge.svg)
+            - [WebdriverIO Visual E2E](./visual-e2e)
```

Comment:
This folder only contains an examples folder. So shouldn't you link to the examples folder?

Secondly, what type of example are run in the examples folder. Should it be put in a folder inside the examples folder that describe what the examples do?
If that is not needed then you might need to change the name `examples` to a more descriptive name (on the other hand, that would be different then then current setup so maybe please put a folder in the `examples` folder :wink: